src/iris/bin/sender.py

```diff
@@ -1010,13 +1010,13 @@ def render(message):
             # format define a generic template that will work for all contexts
             # - even those that are invalid as a final final format maybe repr or pprint
             logger.error(error, message)
-            message['subject'] = '%(message_id)s Iris failed to render your message' % message
+            message['subject'] = 'Iris failed to render your message' % message
```

Comment:
Should we remove the `% message` because we're no longer interpolating any params into the string?

Reply:
don't know how I missed that, done